packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantDishes/RestaurantDish.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+    <div :class="[$style['c-restaurantCard-dish']]">
+        <p :class="[$style['c-restaurantCard-dish-description']]">
+            <span
+                data-test-id="restaurant-dish-name"
+                :class="[$style['c-restaurantCard-dish-name']]">
+                {{ name }}
+            </span>
+            <span
+                v-if="calories"
+                :class="[$style['c-restaurantCard-dish-nutritionalInfo']]">
+                <span
+                    data-test-id="restaurant-dish-calories"
```

Comment:
```suggestion
                    v-if="calories"
                    data-test-id="restaurant-dish-calories"
```

not sure about it but it seems like even if there's portion info and no callories info we don't show this info at all. not sure it it should work this way or not

Reply:
@1itvinka to echo my reply to the other comment - I think for now I am going to leave as is. To my mind, it only makes sense to show portions if there are calories. So if there's no reason for the extra calories check here.

As the requirements haven't been fully explored yet, I'd like to leave this render logic as is for now with the plan to revisit next year once we have a better understanding of the requirements. This is essentially just a starting point for that work :)

Reply:
Yeah, let's see how it goes. Personally, I don't care about calories, but it's good to know the portion size 😂